packages/cli/src/__tests__/project/web3api.buildx.yaml

```diff
@@ -0,0 +1,16 @@
+format: 0.0.1-prealpha.3
+docker:
+  dockerfile: ./Dockerfile
+  buildkit:
+    enabled: true
+    cacheDir: DEFAULT
```

Comment:
I think this should be `cache: string | boolean`, that way users can simply say `cache: true` which will use the default path, or say `cache: ./path/` to specify a specific path.